identity/webapp/next.config.js

```diff
@@ -1,6 +1,9 @@
 const webpack = require('webpack');
 const path = require('path');
-const withTM = require('next-transpile-modules')(['@weco/common', '@weco/catalogue']);
+const withTM = require('next-transpile-modules')([
+  '@weco/common',
+  '@weco/catalogue',
```

Comment:
fwiw I know this was existing but we can/should remove `@weco/catalogue` from here in service of avoiding coupling the apps